conf/messages

```diff
@@ -2827,8 +2828,8 @@ otherReliefs.agent.title=Does your client want to claim any other Capital Gains
 otherReliefs.trust.title=Does the trust want to claim any other Capital Gains Tax reliefs?
 otherReliefs.capacitor.title=Does the person want to claim any other Capital Gains Tax reliefs?
 otherReliefs.personalRep.title=Was the person entitled to claim any other Capital Gains Tax reliefs?
-otherReliefs.personalRepInPeriodOfAdmin.agent.title=Are any other Capital Gains Tax reliefs being claimed?
-otherReliefs.personalRepInPeriodOfAdmin.title=Are any other Capital Gains Tax reliefs being claimed?
+otherReliefs.personalRepInPeriodOfAdmin.agent.title= Are any other reliefs being claimed?
```

Comment:
there's some extra white space at the start of this message and the one below - it's safer to remove them in case they end up getting rendered